docs/upgradeFromRC3.md

```diff
@@ -115,43 +115,36 @@ Time to fetch the `1.0.0` branch and update our `docker-compose` and `composer`
 ```shell
 cd deploy/ec2-docker
 docker-compose down
+git fetch
 git checkout 1.0.0 
 ```
 
-Then copy the appropriate `docker-compose` file for your architecture:
+If you decide to enable the Drupal REDIS module, make sure to add the `REDIS_PASSWORD` variable to your `.env` file.
 
-<!--switch_below
+`IMPORTANT NOTE`: For AWS EC2. If you selected an `IAM role` for your server when setting it up/deploying it, `min.io` will use the AWS EC2-backed internal API to request access to your S3. This means the ROLE itself needs to have read/write access (ACL) to the given Bucket(s) and your key/secrets won't be able to override that. Please do not ignore this note. It will save you a LOT of frustration and coffee. You can also run an EC2 instance without a given IAM and in that case just the ACCESS_KEY/SECRET will matter.
 
-??? info "OSX (macOS)/x86-64"
+Now that you know, you also know that these values **should not be shared** and this `.env` file **should not be committed/kept in version control**. Please be careful.
 
-    ```shell
-    cp docker-compose-osx.yml docker-compose.yml
-    ```
+Then copy the appropriate `docker-compose` file for your architecture:
```

Comment:
@aksm a step is missing here. The user should always backup the original docker-compose.yml. Most of our users end customizing/adding/removing configs and when doing the copy they overwrite all their particularities. Also a git diff example would be great